### DISCUSSION TITLE
blur em parte específica de uma imagem

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#1017] [Perguntas e Respostas]
Eai pessoal beleza? eu to desenvolvendo um app aqui na empresa, e um dos módulos é um foto-upload, onde temos que reconhecer a placa do caminhão e dar um blur nela.
Eu vi diversas libs que dão blur em imagem, o díficil mesmo está fazer o reconhecimento da placa.
Alguem ja passou por um problema parecido? Pelo que andei pesquisando só vi soluções em Python, Java e C#. Queria usar JS.

Nosso APP tá sendo feito em Angular 5. Se souberem de algo também ,me ajudaria muito.
Obrigado.

**Comment by felipefialho** [comment]
De bate pronto pensei no [Resemble.js](https://github.com/HuddleEng/Resemble.js) e no [tracking.js](https://github.com/eduardolundgren/tracking.js), veja se ajudam...

**Comment by rubenmarcus** [comment]
Aqui me disseram no trampo pra fazer um editor de imagem com canvas, e depois salvar a imagem editada via base64 pro banco.. que isso leva pelo menos 1 semana pra fazer

https://trackingjs.com/examples/color_draw_something.html

isso aqui também é uma solução viavel, mas eu ainda nao entendi como posso usa-lo com uma foto mandada pelo usuário e depois salvar a alteração do usuário no banco como a imagem final.
